cmd/nerdctl/network/network_inspect_test.go

```diff
@@ -38,6 +39,128 @@ func TestNetworkInspect(t *testing.T) {
 	)
 
 	testGroup := &test.Group{
+		{
+			Description: "non existent network",
+			Command:     test.RunCommand("network", "inspect", "nonexistent"),
+			// FIXME: where is this error even comin from?
+			Expected: test.Expects(1, []error{errors.New("no network found matching")}, nil),
+		},
+		{
+			Description: "invalid name network",
+			Command:     test.RunCommand("network", "inspect", "∞"),
+			// FIXME: this is not even a valid identifier
```

Comment:
Any reason to avoid ASCII here,?

Reply:
Not really (I just routinely use non-ascii when testing "wrong things" as it tends to wreak more havoc / uncover issues).

If you prefer ASCII, that is fine with me, as long as the identifier is not "valid".

Reply:
The problem here (and the reason for the FIXME here is that identifiers should be validated before looking them up).